contracts/access-controller/Cargo.toml

```diff
@@ -22,6 +22,7 @@ backtraces = ["cosmwasm-std/backtraces"]
 library = []
 
 [dependencies]
+cosmwasm-schema = { version = "1.1.5" }
```

Comment:
Hmm why the need for this? The crate's readme specifically states it's a dev dependency

Reply:
https://book.cosmwasm.com/basics/good-practices.html#json-schema -- just copied over here. It also didn't work as a dev dependency as I wasn't able to use it in schema.rs